llvm/include/llvm/SYCLLowerIR/ComputeModuleProperties.h

```diff
@@ -0,0 +1,45 @@
+//===- ComputeModuleProperties.h - compute properties for module ----------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===---------------------------------------------------------------------===//
+// Functions for computing module properties for SYCL modules.
+//===---------------------------------------------------------------------===//
```

Comment:
```suggestion
//===----------------------------------------------------------------------===//
// Functions for computing module properties for SYCL modules.
//===----------------------------------------------------------------------===//
```

Reply:
thanks for the guidance! fixed in latest commit, thanks.